db/migrate/20190321104441_alter_data_elements.rb

```diff
@@ -9,15 +9,5 @@ def change
     add_column    :data_elements, :academic_year_collected_to,   :integer, default: nil
     add_column    :data_elements, :collection_terms,             :string,  array: true
     add_column    :data_elements, :values,                       :text
-
-    reversible do |dir|
-      dir.up do
-        DataElement.create_translation_table! description: :text
-      end
-
-      dir.down do
-        DataElement.drop_translation_table!
-      end
-    end
```

Comment:
This breaks when applying the migrations to a fresh database because we've removed the translations from the model.

Reply:
Would it be worth to refactor the migrations and remove any reference to `DataElement` translation tables?

Reply:
That's a good idea. We can add a refactor to clean it up further in a separate PR, but I'd like to get this merged as-is as my focus is on getting prod up and running atm.